benchmark/src/jvmMain/kotlin/benchmark/Benchmark.kt

```diff
@@ -0,0 +1,164 @@
+@file:Suppress("unused")
+
+package benchmark
+
+import de.westnordost.osm_opening_hours.model.OpeningHours
+import de.westnordost.osm_opening_hours.parser.toOpeningHoursOrNull
+import dev.sargunv.kotlindsv.Tsv
+import kotlinx.benchmark.Benchmark
+import kotlinx.benchmark.BenchmarkMode
+import kotlinx.benchmark.BenchmarkTimeUnit
+import kotlinx.benchmark.Mode
+import kotlinx.benchmark.OutputTimeUnit
+import kotlinx.benchmark.Scope
+import kotlinx.benchmark.Setup
+import kotlinx.benchmark.State
+import kotlinx.benchmark.TearDown
+import kotlinx.io.buffered
+import kotlinx.io.files.Path
+import kotlinx.io.files.SystemFileSystem
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class InputRecord(
+    @SerialName("?times") val times: String,
```

Comment:
I was confused by this (count vs times), maybe rename to `openingTimes` or `oh`?